cypress/utils/apps/query_enhancements/saved.js

```diff
@@ -433,11 +433,11 @@ const getSavedObjectPostBody = (config, workspaceId, datasourceId, indexPatternI
  * @param {SavedTestConfig} config - the relevant config for the test case
  */
 export const postRequestSaveSearch = (config) => {
-  cy.get('@WORKSPACE_ID').then((workspaceId) => {
-    cy.get('@DATASOURCE_ID').then((datasourceId) => {
-      cy.get('@INDEX_PATTERN_ID').then((indexPatternId) => {
+  return cy.get('@WORKSPACE_ID').then((workspaceId) => {
+    return cy.get('@DATASOURCE_ID').then((datasourceId) => {
+      return cy.get('@INDEX_PATTERN_ID').then((indexPatternId) => {
         // POST a saved search
-        cy.request({
+        return cy.request({
```

Comment:
why `returns`?

Reply:
i wasn't originally seeing things wait for the saved search to load which should be async so i'm returning the promise chain so that then we can chain off the response and then proceed to validate. but i could be missing a wait from intercepting somehting.